src/lang/en.json

```diff
@@ -918,5 +918,12 @@
         }
     },
     "transactionDetailsText": "Transaction Details",
-    "createTokenTextNFT": "Create NFT"
+    "createTokenTextNFT": "Create NFT",
+    "charts": {
```

Comment:
please remove it and use just strings without `i18n`, this page (and task) is just an example and WIP